scripts/nmt/machine_translation.rst

```diff
@@ -13,13 +13,22 @@ Use the following command to train the GNMT model on the IWSLT2015 dataset.
 
 It gets test BLEU score equals to 26.22.
 
-Use the following command to train the Transformer model on the WMT16 dataset for English to German translation. 
+Use the following commands to train the Transformer model on the WMT14 dataset for English to German translation.
 
 .. code-block:: console
 
```

Comment:
Hello, it seems that there is no reference about the paper in this section. Is it possible that you could add a reference so that it is easier for me to write the catalog page?
Thanks!
#1184